test/pylib/manager_client.py

```diff
@@ -86,99 +74,81 @@ async def before_test(self, test_name: str) -> None:
         """Get number of configured replicas for the cluster (replication factor)"""
-        resp = await self._get_text("/cluster/replicas")
+        resp = await self.cli.get_text("/cluster/replicas")
         return int(resp)
 
     async def servers(self) -> List[str]:
         """Get list of running servers"""
-        host_list = await self._get_text("/cluster/servers")
-        return host_list.split(',')
+        host_list = await self.cli.get_text("/cluster/servers")
+        return host_list.split(",")
 
     async def mark_dirty(self) -> None:
         """Manually mark current cluster dirty.
            To be used when a server was modified outside of this API."""
-        await self._get_text("/cluster/mark-dirty")
+        await self.cli.get_text("/cluster/mark-dirty")
 
     async def server_stop(self, server_id: str) -> None:
         """Stop specified server"""
         logger.debug("ManagerClient stopping %s", server_id)
-        await self._get_text(f"/cluster/server/{server_id}/stop")
+        await self.cli.get_text(f"/cluster/server/{server_id}/stop")
 
     async def server_stop_gracefully(self, server_id: str) -> None:
         """Stop specified server gracefully"""
         logger.debug("ManagerClient stopping gracefully %s", server_id)
-        await self._get_text(f"/cluster/server/{server_id}/stop_gracefully")
+        await self.cli.get_text(f"/cluster/server/{server_id}/stop_gracefully")
 
     async def server_start(self, server_id: str) -> None:
         """Start specified server"""
         logger.debug("ManagerClient starting %s", server_id)
-        await self._get_text(f"/cluster/server/{server_id}/start")
+        await self.cli.get_text(f"/cluster/server/{server_id}/start")
         self._driver_update()
 
     async def server_restart(self, server_id: str) -> None:
         """Restart specified server"""
         logger.debug("ManagerClient restarting %s", server_id)
-        await self._get_text(f"/cluster/server/{server_id}/restart")
+        await self.cli.get_text(f"/cluster/server/{server_id}/restart")
         self._driver_update()
 
     async def server_add(self) -> str:
         """Add a new server"""
-        server_id = await self._get_text("/cluster/addserver")
+        server_id = await self.cli.get_text("/cluster/addserver")
         self._driver_update()
         logger.debug("ManagerClient added %s", server_id)
         return server_id
 
     async def server_remove(self, server_id: str) -> None:
         """Remove a specified server"""
         logger.debug("ManagerClient removing %s", server_id)
-        await self._get_text(f"/cluster/removeserver/{server_id}")
+        await self.cli.get_text(f"/cluster/removeserver/{server_id}")
         self._driver_update()
 
-    async def server_get_config(self, server_id: str) -> dict[str, object]:
-        resp = await self._get(f"/cluster/server/{server_id}/get_config")
+    async def server_get_config(self, server_id: str) -> Dict[str, object]:
```

Comment:
Upgrade your mypy/python version if `dict` doesn't work for you.

Reply:
reverted in https://github.com/scylladb/scylladb/commit/7cc3aedb8d083a86de5cf4d92fd4c3190af4413d